src/main/java/frc/robot/command_factories/LEDFactory.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.command_factories;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+
+public class LEDFactory {
+  public static void init() {
+    RobotContainer.getLed()
+        .setColorVoid(
+            Constants.LEDConstantsLeonidas.startColor, Constants.LEDConstantsLeonidas.endColor);
+  }
+
+  public static Command solid() {
```

Comment:
I think its OK to have 'default' start and end colors of white and red.  But - i think we are limiting ourselves by not allowing the 'calling function' to override the 'blink' and 'solid' colors to something else if needed.  ie. solid green to indicate the starting position of the robot is correct.   Perhaps impement an override of solid() and blink() methods to allow them to accept a starting/ending color.